tests/unit/ui/renderers/test_panels.py

```
@@ -0,0 +1,232 @@
+
+    def test_returns_tuple_of_three(self) -> None:
+        result = _truncate_content(
+            SMALL_CONTENT,
+            max_lines=10,
+            max_line_width=DEFAULT_MAX_LINE_WIDTH,
+        )
+        assert isinstance(result, tuple)
+        assert len(result) == 3
+
+    def test_empty_content(self) -> None:
+        content, shown, total = _truncate_content(
```

Ruff (0.14.14)

[warning] Unpacked variable `content` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)